setup.py

```diff
@@ -275,7 +275,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'sphinx-argparse>=0.1.13',
     'sphinx-autoapi>=1.8.0',
     'sphinx-copybutton',
-    'sphinx-jinja>=1.1',
+    'sphinx-jinja>=1.1, <2.0',
```

Comment:
```suggestion
    # we can remove the upper limit when we will apply the fix as explained in the change log:
    # https://github.com/tardyp/sphinx-jinja/blob/d4f3ba9aa2b6a1217dfd90c676475d516e69b306/ChangeLog#L8-L16
    'sphinx-jinja>=1.1, <2.0',
```

nit: I think we should comment why we have an upper bound

Reply:
Much better than mine.